sqlite3_opt_dbstat_vtab.go

```diff
@@ -0,0 +1,9 @@
+// +build sqlite_dbstat_vtab dbstat_vtab
+
+package sqlite3
+
+/*
+#cgo CFLAGS: -DSQLITE_ENABLE_DBSTAT_VTAB=1
+#cgo LDFLAGS: -lm
```

Comment:
If it required to link `libm` to enable the DBSTAT table?

Reply:
You're right.